pkgs/tools/audio/beets/default.nix

```
@@ -284,7 +284,7 @@ in pythonPackages.buildPythonApplication rec {
     description = "Music tagger and library organizer";
     homepage = "http://beets.io";
     license = licenses.mit;
-    maintainers = with maintainers; [ aszlig domenkozar pjones ];
+    maintainers = with maintainers; [ aszlig domenkozar lovesegfault pjones ];
```

Please add me as well :+1: 

```suggestion
   maintainers = with maintainers; [ aszlig domenkozar lovesegfault doronbehar pjones ];
```

---

Done.